lib/svg-preview-view.js

```diff
@@ -27,6 +27,17 @@ class SvgPreviewView extends View {
             this.text('transparent')
           })
         })
+        this.div({ class: 'image-controls-group btn-group' }, () => {
+          this.a({ class: 'btn', outlet: 'zoomOutButton' }, () => {
```

Comment:
This needs to be a button instead of an anchor.

```
this.button({ class: 'btn', outlet: 'zoomOutButton' }, () => {})
```
<img width="116" alt="screen shot 2016-09-20 at 13 40 21" src="https://cloud.githubusercontent.com/assets/423234/18668789/dafb4b8a-7f37-11e6-8b98-8cddc50ff015.png">